api/src/main/java/io/strimzi/api/kafka/model/connect/KafkaConnectTemplate.java

```diff
@@ -181,6 +183,26 @@ public void setClusterRoleBinding(ResourceTemplate clusterRoleBinding) {
         this.clusterRoleBinding = clusterRoleBinding;
     }
 
+    @Description("Template for the Kafka Connect Role")
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    public ResourceTemplate getRole() {
+        return role;
+    }
+
+    public void setRole(ResourceTemplate role) {
+        this.role = role;
+    }
+
+    @Description("Template for the Kafka Connect RoleBinding")
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    public ResourceTemplate getRoleBinding() {
+        return roleBinding;
+    }
+
+    public void setRoleBinding(ResourceTemplate roleBinding) {
+        this.roleBinding = roleBinding;
+    }
```

Comment:
Very stupid question but I didn't get why we need this role and roleBinding field in the template (which is supposed to be something written by the users). Aren't we generating role and rolebinding programmatically for accessing Secrets?